base/src/main/java/com/smartdevicelink/proxy/rpc/OnVehicleData.java

```diff
@@ -1,885 +1,885 @@
-/*
```

Comment:
@kostyaBoss It looks like the entire file `OnVehicleData.java` is being regenerated by the code generator tool.  Can you just merge in the changes for this PR (data related to `seatOccupancy`)?